release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -453,6 +453,32 @@ installation does not fail {product-title} cluster installation.
 You can xref:../openshift_images/samples-operator-alt-registry.adoc#installation-restricted-network-samples_samples-operator-alt-registry[configure alternate or mirrored registries]
 to bypass this issue if it arises during your cluster installation.
 
+[id="ocp-4-4-documentation-conventions"]
+=== Documentation conventions
+
+[id="ocp-4-4-openshift-documentation-licensed-under-apache-2-0"]
+==== OpenShift documentation licensed under Apache license 2.0
+
+The OpenShift documentation is now licensed under
+link:https://www.apache.org/licenses/LICENSE-2.0[Apache license 2.0]. It was
+previously licensed under the Creative Commons Attribution-ShareAlike 3.0	
+Unported license.
+
+[id="ocp-4-4-copy-button-for-docs-openshift-com-site"]
+==== Copy button for docs.openshift.com site
+
+All code blocks on docs.openshift.com now provide a Copy button that lets you
+copy all text in a code block to your machine's clipboard. The Customer Portal
+will not have this capability.
```

Comment:
This capability is not available on the customer portal version of OpenShift docs.